asmcomp/closure.ml

```diff
@@ -1080,12 +1083,16 @@ and close_functions fenv cenv fun_defs =
     List.flatten
       (List.map
          (function
-           | (id, Lfunction{kind; params; body}) ->
-               split_default_wrapper id kind params body
+           | (id, Lfunction{kind; params; body; attr}) ->
+               split_default_wrapper id kind params body attr
            | _ -> assert false
          )
          fun_defs)
   in
+  let inline_attribute = match fun_defs with
+    | [_, Lfunction{kind; params; body; attr = { inline }}] -> inline
+    | _ -> false (* recursive functions can't be inlined *)
```

Comment:
That's the quick and dirty way for closure. It will gets proper treatment on flambda.